drivers/scheduler/openshift/openshift.go

```diff
@@ -327,6 +328,29 @@ func (k *openshift) createTorpedoSecurityContextConstraints() (*ocpsecurityv1api
 	return torpedoScc, err
 }
 
+func updateMaxUnavailableWorkerMCP() error {
+
+	maxUnavailable := os.Getenv("OCP_SURGE_UPGRADE_VALUE")
+	log.Info("Setting maxUnavailable value for worker MCP to [%s]", maxUnavailable)
+	patchData := fmt.Sprintf(`{"spec":{"maxUnavailable":"%s"}}`, maxUnavailable)
+
+	t := func() (interface{}, bool, error) {
+		var output []byte
+		args := []string{"patch", "mcp", "worker", "--type=merge", "--patch", patchData}
```

Comment:
Does this update happens instantly? You sure we don't need to check that it actually finished updating it? I also would recommend to use API rather then CLI, I think it is better for Torpedo, since we mostly use API everywhere, unless it is not possible

Reply:
The only method I could find as per Redhat is CLI - https://access.redhat.com/solutions/5453781
Even the admin acks are CLI which are called before starting the upgrade. 

I will add a check to validate if the MCP is patched.